hardemu/keyboard.js

```diff
@@ -296,6 +296,13 @@ function Keyboard(Zeal, PIO) {
         return true;
     }
 
+    function key_pressstr(str) {
```

Comment:
That's not a key press anymore, that's a key sequence :smile:

Reply:
Renamed.